metadata.json

```
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-device_manager",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

we should not bump this before the actual release. bumping the version is part of the release process.
```suggestion
  "version": "3.1.0",
```